qubes/app.py

```diff
@@ -1404,6 +1404,24 @@ def _get_pool(**kwargs):
             raise qubes.exc.QubesException('No driver %s for pool %s' %
                                            (driver, name))
 
+    async def stop_storage(self, loop):
+        """
+        Stop the storage of all domains that are not running.
+        """
+        async def stop(i):
+            async with i.startup_lock:
+                if not i.is_running():
+                    await i.storage.stop()
+        future = (qubes.utils.void_coros_maybe(stop(i)
```

Comment:
Since `stop()` is always an async function, I think you don't need `void_coros_maybe` helper here.

Reply:
Good idea; this also makes the code cleaner, since `asyncio.wait` takes an iterable already.